test/conftest.py

```diff
@@ -42,6 +42,15 @@
 from pudl.workspace.datastore import Datastore
 from pudl.workspace.setup import PudlPaths
 
+pudl.logging_helpers.configure_root_logger(
+    dependency_loglevels={
+        "numba": logging.DEBUG,
+        "fsspec": logging.INFO,
+        "asyncio": logging.INFO,
+        "google": logging.INFO,
+        "alembic": logging.WARNING,
```

Comment:
Nice to be able to set these independently :)

I'm non-blockingly curious about a couple of these:
* we were setting `numba` to `WARNING` before, why the downgrade to `DEBUG`?
* for `fsspec`, `asyncio`, and `google`, were they somehow being set to some non-`INFO` level in some other place?

Reply:
Ah yes `numba` should be `WARNING`, fixed